4-WebApp-your-API/4-2-B2C/TodoListService/Startup.cs

```diff
@@ -61,6 +62,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHttpsRedirection();
 
             app.UseRouting();
+            JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
```

Comment:
instead of calling this here, can we uncomment the line 31 above? Would also appreciate if you could update the comments there (feel free to reference the discussion [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/415))